db/migrate/20220720214646_add_delete_cascade_from_orgs_down.rb

```diff
@@ -0,0 +1,11 @@
+class AddDeleteCascadeFromOrgsDown < ActiveRecord::Migration[7.0]
+  def up
+    remove_foreign_key :schemes, :organisations, column: "owning_organisation_id"
+    remove_foreign_key :schemes, :organisations, column: "managing_organisation_id"
+    add_foreign_key :schemes, :organisations, column: "owning_organisation_id", on_delete: :cascade
+  end
+
+  def down
+    remove_foreign_key :schemes, :organisations, column: "owning_organisation_id"
```

Comment:
This down isn't quite the opposite of up - it won't get you back to your original state because originally you did have a foreign key, just without the delete cascade option

Reply:
Should be fixed now